go.mod

```diff
@@ -1,11 +1,11 @@
-go 1.16
+go 1.17
 
 module github.com/terra-money/core
 
 require (
 	github.com/CosmWasm/wasmvm v0.16.1
-	github.com/cosmos/cosmos-sdk v0.44.2
-	github.com/cosmos/ibc-go v1.1.0
+	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/cosmos/ibc-go v1.2.4
```

Comment:
We made new branch for next update `release/v0.6.x` and it will be good to update ibc-go version to that branch.

Reply:
great idea!

I shouldn't have changed the version number there, anyhow.  

If we want to target wasm 1.0, we should also plan to update to sdk v0.45.0